DearDiarySwift/Assets.xcassets/logo.imageset/download.svg

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
- [ ] SVG's can have issues with older versions of iOS. But it uses less space and looks crisp if you use SVGs. So, whenever you use SVGs, you'll have to test in the oldest version of iOS, that your app supports to make sure that it works without any issue